### DISCUSSION TITLE
Does box64+ wine compatible with GALLIUM_HUD=

### DISCUSSION THREAD

**Discussion by xDoge26** [ptitSeb/box64#440] [Q&A]
can i use it

**Comment by ptitSeb** [comment]
They should be no issue with it. The libGL used is your native mesa version, so it should work the same way as if you were running any other software.

**Reply by xDoge26** [reply]
> They should be no issue with it. The libGL used is your native mesa version, so it should work the same way as if you were running any other software.

thanks
